### DISCUSSION TITLE
Default Locale in "Localization" and "Edit message bundles" discrepancy

### DISCUSSION THREAD

**Discussion by mpalavrov** [keycloak/keycloak#24014] [Archive]
The default language under "Edit message bundles" is always **English**, even if under Localization different is set, like **Deutsch**
![image](https://github.com/keycloak/keycloak/assets/33352511/e05c5109-a917-4016-bbc6-16ace898a95a)

### Steps to reproduce:

- Go to Realm Settings --> Localization
- Enable **Internationalization**
- Add multiple languages (doesn't matter if English is included)
- Select a language (different than English) for Default Locale
![image](https://github.com/keycloak/keycloak/assets/33352511/83d6f294-06de-46f2-9f80-5d8703c860ad)

#### Expected behaviour
Under **Edit message bundles** to have only the selected "Supported Locales" and the Default language to be the same as selected above

#### Actual behaviour
The Default Locale under **Edit message bundles** is "English, even if not selected as Supported Locale (and for sure as Default):
![image](https://github.com/keycloak/keycloak/assets/33352511/dc85969d-35fe-46fb-9f3e-24259d0fc9a4)

----

As for the Fallback locale, it is falling back to English only, probably this is the reason to be displayed this way under the "Edit message bundles", but if this is true, better description is needed in the UI somehow, so there are no confusions.

**Comment by jonkoops** [comment]
@mpalavrov since this is a bug, could you report it as such on our issue tracker?

**Comment by mpalavrov** [comment]
Hi @jonkoops,
Thank you for checking. I have raised one in the issue tracker - https://github.com/keycloak/keycloak/issues/24239
Best,
Milen
